src/app/search-form/search-form.component.html

```diff
@@ -0,0 +1,75 @@
+<div class="autocomplete-section">
+
+  <form [formGroup]="autocompleteForm" (ngSubmit)="onSubmit()">
+    <app-autocomplete
+      [placeholder]="placeholder"
+      [sourceArray]="citiesNames"
+      [customClass]="customClass"
```

Comment:
It would be better to use 1 input for autocomplete options instead of several. Just use input and send object with appropriate settings